cue.c

```diff
@@ -254,7 +257,6 @@ static int cue_read_comments(struct input_plugin_data *ip_data, struct keyval **
 	Track *t;
 	Rem *track_rem;
 	Cdtext *track_cdtext;
-	char *val;
```

Comment:
@flyingmutant can probably say for certain, but my guess is that the previous forward declarations of these variables was to maintain C90 compatibility. I have no idea if this is a priority or not, but you can achieve this with less churn in this diff by just marking this as `const` here.

Apologies for butting in on this PR if C90 compatibility is not relevant.